enterprise/api/v1/views.py

```diff
@@ -525,6 +526,39 @@ def unlink_users(self, request, pk=None):  # pylint: disable=unused-argument
 
         return Response(status=HTTP_200_OK)
 
+    @action(detail=False)
+    def algolia_key(self, request, *args, **kwargs):
+        """
+        Returns an Algolia API key that is secured to only allow searching for
+        objects associated with enterprise customers that the user is linked to.
+        """
+
+        if not settings.ENTERPRISE_ALGOLIA_SEARCH_API_KEY:
```

Comment:
```suggestion
        if not (api_key := getattr(settings, "ENTERPRISE_ALGOLIA_SEARCH_API_KEY", "")):
```

This way, we will make this compatible with all releases without any changes to the `edx-platform`.